docs/source/membership/membership.md

```diff
@@ -1,341 +1,197 @@
-identities from the `ORG1-MANUFACTURING` OU might be able to access a channel,
-whereas `ORG1-DISTRIBUTION` cannot.
-
-Finally, though this is a slight misuse of OUs, they can sometimes be used by
-different organizations in a consortium to distinguish each other. In such cases, the
-different organizations use the same Root CAs and Intermediate CAs for their chain
-of trust, but assign the `OU` field to identify members of each organization.
-We'll also see how to configure MSPs to achieve this later.
-
-### Local and Channel MSPs
-
-MSPs appear in two types of places in a blockchain network:
+# Membership Service Provider (MSP)
+
+## Why do I need an MSP?
+
+Because Fabric is a permissioned network, blockchain participants need a way to prove their identity to the rest of the network in order to transact on the network. If you've read through the documentation on [identity](../identity/identity.html)
+you've seen how a PKI can provide verifiable identities through a chain of trust. How is that chain of trust used by the blockchain network? Recall from the credit card scenario in the Identity topic that the Certificate Authority is like a card
+provider — it dispenses many different types of verifiable identities. An MSP, on the other hand, determines which credit card providers are accepted at the store. In this way, the MSP turns an identity (the credit card) into a role (the ability to buy things at the store).
+
+This ability to turn verifiable identities into roles is fundamental to the way Fabric networks function, since it allows organizations, nodes, and channels the ability establish MSPs that determine who is allowed to do what at the organization, node, and channel level.
+
+![MSP1a](./membership.msp.diagram.png)
+
+*Identities are similar to your credit cards that are used to prove you can pay. The MSP is similar to the list of accepted credit cards.*
+
+Or consider a consortium of banks that operate a blockchain network. Each bank operates peer and ordering nodes, and the peers endorse transactions submitted to the network. However, each bank would also have departments and account holders. The account holders would belong to each organization, but would not run nodes on the network. They would only interact with the system from their mobile or web application. So how does the network recognize and differentiate these identities? MSP's are used to define the organizations that belong to the network, whose members are trusted by other network members. MSPs are also the mechanism that provide members with a set of roles and permissions within the network.
+
+Finally, consider if you want to join an _existing_ network, you need a way to turn your identity into something that is recognized
+by the network. The MSP is the mechanism that enables you to participate on a permissioned blockchain network. To transact on a Fabric network a member needs to:
+
+- Have an identity issued by a CA that is trusted by the network.
```

Comment:
I agree with Nik. There's not enough conceptual separation. Might be better as an ordered list rather than a conceptual list. Something like:

To transact on a Fabric network a member needs to:

1. Have an identity issued by a CA trusted by the network. 
2. Either create an MSP or have this identity added as an admin of the organization. 
3. Have the MSP added as a trusted member of the network. 
4. Make sure the configuration of the network includes this MSP. 

Then a link in #4 to the relevant section in the policies doc or updating a channel doc about how this is achieved.